.github/workflows/push.yml

```diff
@@ -7,25 +7,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2.3.4
-        with:
-          fetch-depth: 0
-
-      - name: Use Node.js 14.x
-        uses: actions/setup-node@v2.4.0
-        with:
-          node-version: 14.x
-          registry-url: https://registry.npmjs.org
-          cache: 'yarn'
-
-      - name: Install
-        run: yarn
-
-      - name: Build
-        run: yarn run build
-
-      - name: Lint
-        run: yarn run lint
+      - uses: linz/action-typescript@v1
```

Comment:
Would it be possible to make the Node.js version a mandatory configuration item (or default it to something like `latest-lts`)?

Reply:
The goal is to make it when a new LTS is released to upgrade all the pipelines to the new LTS and then see what breaks :D 

Generally going between node versions causes no issues unless your using c libraries which not many people do.